pkg/api/validation/validation.go

```diff
@@ -58,6 +58,8 @@ var DNS1123LabelErrorMsg string = fmt.Sprintf(`must be a DNS label (at most %d c
 var DNS952LabelErrorMsg string = fmt.Sprintf(`must be a DNS 952 label (at most %d characters, matching regex %s): e.g. "my-name"`, validation.DNS952LabelMaxLength, validation.DNS952LabelFmt)
 var pdPartitionErrorMsg string = IntervalErrorMsg(0, 255)
 var PortRangeErrorMsg string = IntervalErrorMsg(0, 65536)
+var GroupIdRangeErrorMsg string = IntervalErrorMsg(-1, 2147483648)
```

Comment:
Actually, let's do this:

// TODO: once we have a type for UID/GID we should make these that type.
const minUserID = 0
const maxUserID = math.MaxInt32
const maxGroupID = 0
const maxGroupID = math.MaxInt32